release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -778,6 +778,15 @@ For more information, see xref:../updating/updating-restricted-network-cluster/r
 
 In {product-title} {product-version}, the `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster. This service determines the network interface controller (NIC) that {product-title} uses for communication with the Kubernetes API server when the node boots. In rare circumstances, the service might select an incorrect node IP after an upgrade. If this happens, you can use the `NODEIP_HINT` feature to restore the original node IP, as described in the "Troubleshooting network issues" section of the documentation.
 
+[discrete]
+[id="ocp-4-13-rhcos-symbolic-disk-naming-change"]
+=== It is now recommended to use persistent names for device parameters
+{product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. With this update, symbolic disk naming can change across reboots. This can cause {op-system} installation issues after first boot in the case that the sysroot directory was installed on a device such as `/dev/sda`. It is recommended to use a specific naming scheme for devices, including for any specific disk references, such as `dev/disk/by-id`.
+
```

Comment:
I discussed this issue with @travier and others on the team yesterday, 

> This can cause {op-system} installation issues after first boot in the case that the sysroot directory was installed on a device such as `/dev/sda`.

I think this statement may need to be corrected. The OS disk/sysroot is not affected by any subsequent changes to the device naming, as it doesn't rely on these names to boot. 
The risk is around non-OS disks that might be defined/referenced by the user either at time of node provisioning or post-install. For example, if the user applies an install manifest/MachineConfig that references a /dev/sdb in the config for some service (fstab, custom systemd service/unit, etc.), then there is a risk the device name will change on a subsequent reboot. Operators that  The effects will be dependent on what component was being (re)configured.

Reply:
thanks @control-d ! so would it make sense to say something like:

> This can cause issues if you apply configuration files after installation or when provisioning a node that reference a disk which uses symbolic naming, such as `/dev/sda`, for creating services. The Operators that this affects are dependent on the particular component you are configuring.